projects/ion/src/lib/alert/alert.component.scss

```diff
@@ -0,0 +1,63 @@
+@import '../../styles/index.scss';
+
+@mixin alert-style($backgroud, $borderColor) {
+  background-color: $backgroud;
+  border-color: $borderColor !important;
+
+  .alert-icon {
+    ::ng-deep svg {
```

Comment:
Icons are not center aligned to text, you can add `display: flex` and `align-items: center` here to fix it.